### ISSUE TITLE
Hyperparameter graph: val_loss as metric

### ISSUE THREAD

**Issue by martsalz** [microsoft/nni#1483]
**Short summary about the issue/question**:

If `val_loss` is specified as a metric in the python code and top 20 are selected in the hyperparameter graph as in the example below, I actually expect that the experiments with the lowest `val_loss` values will be displayed. In fact, the experiments with the highest `val_loss` values are plotted. It seems that the selection is only intended for the `accuracy` metric.

![1](https://user-images.githubusercontent.com/12211578/63406110-2e1d9e00-c3e9-11e9-8b05-f6f740118ea7.PNG)

![2](https://user-images.githubusercontent.com/12211578/63406114-2fe76180-c3e9-11e9-8b23-714dd8f6525b.PNG)

**Comment by martsalz** [comment]
Therefore, with `val_loss` as metric, the colors of the experiments have to be chosen differently. For instance, the experiment with the lowest `val_loss` in red color.

**Comment by scarlett2018** [comment]
Thanks @martsalz for raising the request. You are right that the current hyperparameter graph design is optimized for default metrics like accuracy. We need a design for supporting metrics like val_loss. 

@rabbit008 will work on the design proposal first. @martsalz - Any suggestions on the design idea are appreciated.

**Comment by ajoeajoe** [comment]
is it support logloss on NNI？

**Comment by martsalz** [comment]
The Keras or e.g. Tensorflow code is minimally modified and the metric is specified in the Python code. Therefore NNI can execute everything that works under Keras/Tensorflow.

**Comment by ultmaster** [comment]
I agree that current design for multiple metrics is really weak.

**Comment by ultmaster** [comment]
Also tracked in #1202.

**Comment by Lijiaoa** [comment]
@martsalz  PR #2586 had support `val_loss as metric`. 
@kvartet Could you help close this issue?
